src/main/java/core/basesyntax/FigureSuplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSuplier {
+    public static final int FIGURE_COUNT = 4;
+    private Random random = new Random();
+    private ColorSupplier colors = new ColorSupplier();
```

Comment:
bad variable name colors for a developer means list of objects with type Color, but it is not true in your case